daprdocs/content/en/developing-applications/ides/intellij.md

```diff
@@ -34,7 +71,8 @@ Create or edit the file in `$HOME/.IdeaIC2019.3/config/tools/External\ Tools.xml
   <tool name="dapr for DemoService in examples" description="Dapr sidecar" showInMainMenu="false" showInEditor="false" showInProject="false" showInSearchPopup="false" disabled="false" useConsole="true" showConsoleOnStdOut="true" showConsoleOnStdErr="true" synchronizeAfterRun="true">
     <exec>
       <!-- 2. For Linux or MacOS use: /usr/local/bin/daprd -->
-      <option name="COMMAND" value="C:\dapr\daprd.exe" />
+      <!-- 2.1 Replace %USERPROFILE% with the full path -->
+      <option name="COMMAND" value="%USERPROFILE%\dapr\daprd.exe" />
```

Comment:
Lets discourage users from using daprd directly. `dapr` already supports running sidecar only.

Reply:
Line 18 covers calling dapr and starting a sidecar vs setting up debugging here.  Can `dapr` be called here as well?

Reply:
done